server/venueless/api/views.py

```diff
@@ -82,6 +86,26 @@ def patch(self, request, **kwargs):
         return Response(serializer.data)
 
 
+class WorldThemeView(APIView):
+
+    permission_classes = []
+
+    def get(self, request, **kwargs):
+        """
+        Retrieve theme config of a world
+        @param request: request obj
+        @param kwargs: world_id
+        @return: theme data of a world
+        """
+        try:
+            world = get_object_or_404(World, id=kwargs["world_id"])
+            return Response(WorldSerializer(world).data['config']['theme'])
+        except KeyError:
+            logger.error(f"error happened when trying to get theme data of world: " + kwargs["world_id"])
```

Comment:
Incorrect use of logger. Check guide in https://github.com/fossasia/eventyay-talk/pull/130